llvm/lib/Target/AIE/AIEHazardRecognizer.h

```diff
@@ -26,6 +26,9 @@
 #include "llvm/MC/MCInstrItineraries.h"
 
 namespace llvm {
+// Bit 0-3 if for bank A, B, C, D respectively
+// Bit 4 if for Tile Memory
```

Comment:
Nit: This comment will get out of date, do we need it?

Reply:
Will remove it, kept it for improving readability. You point is valid it might get out of date.